openbook_posts/tests/views/test_post.py

```diff
@@ -1301,6 +1301,107 @@ def test_post_comment_notification_is_deleted_when_deleting_comment(self):
         self.assertFalse(PostCommentNotification.objects.filter(pk=post_comment_notification.pk).exists())
         self.assertFalse(Notification.objects.filter(pk=notification.pk).exists())
 
+    def test_can_edit_own_post_comment_on_own_post(self):
+        """
+            should be able to edit own post comment
+        """
+
+        user = make_user()
+        post = user.create_public_post(text=make_fake_post_text())
+        original_post_comment_text = make_fake_post_comment_text()
+        post_comment = user.comment_post_with_id(post.pk, text=original_post_comment_text)
+
+        url = self._get_url(post_comment=post_comment, post=post)
+
+        edited_post_comment_text = make_fake_post_comment_text()
+        headers = make_authentication_headers_for_user(user)
+
+        response = self.client.patch(url, {
+            'text': edited_post_comment_text
+        }, **headers)
+
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        post_comment_db = PostComment.objects.get(pk=post_comment.pk)
```

Comment:
you can use `post_comment.db_refresh(`) or something like that to refresh the model with db data.